Assets/NuGet/Editor/NugetHelper.cs

```diff
@@ -1251,11 +1251,18 @@ public static Stream RequestUrl(string url, string password, int? timeOut)
                 getRequest.ReadWriteTimeout = timeOut.Value;
             }
 
-            if (packageHost.EndsWith("pkgs.visualstudio.com") && string.IsNullOrEmpty(password))
+            if (string.IsNullOrEmpty(password))
             {
-                // The host is a VisualStudio feed (which requires authentication) but a password was not provided. Use the VSS credential provider to aquire a token and append
-                // it to the request.
-                password = GetPasswordFromVSTSCredentialProvider(packageHost);
+                bool isKnownAuthenticatedFeed =
+                    packageHost.EndsWith("pkgs.visualstudio.com") ||
+                    packageHost.EndsWith("pkgs.dev.azure.com");
```

Comment:
Could we just put a checkbox on the feed in the settings to say "Use Credential Manager"?

It's a suggestion, not a blocker :)

Reply:
Really like this suggestion, added it as an enhancement #245 . I think I'll hold off on biting that off right now though. =)